src/components/blipLoading/blipLoading.component.ts

```diff
@@ -0,0 +1,37 @@
+import angular from 'core/angular';
+import { BlipLoading } from 'blip-toolkit';
+import * as uuid from 'uuid';
+import { IComponentController} from 'angular';
+
+const BLIP_LOADING_PREFIX = 'blip-loading-';
+
+class BlipLoadingController implements IComponentController {
+    type: string;
```

Comment:
Use union type to declare wich loading type is possible: `type: 'dotted' | 'solid';`

Reply:
nice! while I'm using it, does `'dotted` become the default value? If yes, I can remove `|| 'dotted'`
from here: 
` const instanceOptions = {
        type: this.type || 'dotted',
  };`
right?

Reply:
Nope! You can define default value like this: `type: 'dotted' | 'solid' = 'dotted'`;